src/client.js

```diff
@@ -58,20 +58,14 @@ class StitchClient {
   }
 
   /**
-   * Login to stich instance, optionally providing a username and password. In
```

Comment:
Why remove the jsdoc?

Reply:
Oversight, originally I had this function entirely removed before realizing the implications of removing a client-level method. Going to add it back and re-word accordingly.